clientv3/retry.go

```diff
@@ -86,12 +87,13 @@ func (c *Client) newRetryWrapper(isStop retryStopErrFunc) retryRpcFunc {
 func (c *Client) newAuthRetryWrapper() retryRpcFunc {
 	return func(rpcCtx context.Context, f rpcFunc) error {
 		for {
+			pinned := c.balancer.pinned()
 			err := f(rpcCtx)
 			if err == nil {
 				return nil
 			}
 			if logger.V(4) {
-				logger.Infof("clientv3/auth-retry: retry for error %v", err)
+				logger.Infof("clientv3/auth-retry: retry for error %v (%s)", err, pinned)
```

Comment:
retry on pined endpoint (%s) due to error %v

Reply:
@xiang90 That endpoint is errored, so it's unpinned by this line. So it's possible that we retry on different endpoint?

Reply:
or let's say ` error %s on pined endpoint (%s))

basically only (%s) is not descriptive.